lib/game.rb

```diff
@@ -0,0 +1,57 @@
+require_relative './board.rb'
+
+class Game
+  def initialize(arr)
+    @table = arr
+  end
+
+  def self.valid_letter?(letter)
+    return true if %w[X O].include?(letter)
+  end
+
+  def valid_entry?(location)
+    return true if @table[location] == ' '
+
+    false
+  end
+
+  attr_accessor :player1_name, :player2_name, :player1_letter, :player2_letter
```

Comment:
- [ ] use `attr_reader/attr_writer` instead of `attr_accessor`. This will prevent someone from modifying the behavior of these variables from outside the class which may present security risk.